php/Plugin.php

```diff
@@ -88,11 +89,77 @@ public function require_deprecated() {
 		}
 	}
 
+	/**
+	 * Require the API functions.
+	 */
+	private function require_api() {
+		require_once __DIR__ . '/BlockApi.php';
+	}
+
 	/**
 	 * Requires helper functions.
 	 */
 	private function require_helpers() {
-		require_once __DIR__ . '/Helpers.php';
-		require_once __DIR__ . '/BlockApi.php';
+		if ( $this->is_plugin_conflict() ) {
+			add_action( 'admin_notices', [ $this, 'plugin_conflict_notice' ] );
+		} else {
+			require_once __DIR__ . '/Helpers.php';
+		}
+	}
+
+	/**
+	 * Gets whether there is a conflict from another plugin having the same functions.
+	 *
+	 * @return bool Whether there is a conflict.
+	 */
+	public function is_plugin_conflict() {
+		return function_exists( 'block_field' ) || function_exists( 'block_value' );
+	}
+
+	/**
+	 * An admin notice for another plugin being active.
+	 */
+	public function plugin_conflict_notice() {
+		$plugin_file      = 'block-lab/block-lab.php';
+		$deactivation_url = add_query_arg(
+			[
+				'action'        => 'deactivate',
+				'plugin'        => rawurlencode( $plugin_file ),
+				'plugin_status' => 'all',
+				'paged'         => 1,
+				'_wpnonce'      => wp_create_nonce( 'deactivate-plugin_' . $plugin_file ),
+			],
+			admin_url( 'plugins.php' )
+		);
+
+		?>
+		<style>
+			.gcb-notice-conflict {
+				display: flex;
+				padding-top: 0.2rem;
+				padding-bottom: 0.2rem;
+			}
+
+			.gcb-notice-conflict .gcb-conflict-copy {
+				margin-right: auto;
+			}
+
+			.gcb-notice-conflict .gcb-link-deactivate {
+				margin: auto 0.2rem;
+			}
+		</style>
+		<div id="gcb-conflict-notice" class="notice notice-error gcb-notice-conflict">
+			<div class="gcb-conflict-copy">
+				<p><?php esc_html_e( 'It looks like Block Lab is active. Please deactivate it or migrate, as it will not work while Genesis Custom Blocks is active.', 'genesis-custom-blocks' ); ?></p>
```

Comment:
Hey Ryan, I know you're still working on this but wanted to mention it while it's on my mind. This text won't make sense if the version of BL installed doesn't have the migration code in it.

Reply:
Ah, good point.

Reply:
Great idea, applied in https://github.com/studiopress/genesis-custom-blocks/pull/29/commits/b0da132e0f71e801c85aee10da99eafaa39df906